app/views/static/default_landing/partials/_structured_text.html.erb

```
@@ -0,0 +1,17 @@
+<% raise 'Could not find structured text content' if 
+(local_assigns['icon_color'] && local_assigns['icon'] && local_assigns['header'] && local_assigns['text']).nil? %>
+
+<h2 class="Vlt-title--icon">
+    <svg class="Vlt-<%= local_assigns['icon_color'] %>"><use xlink:href="/symbol/volta-icons.svg#Vlt-<%= local_assigns['icon'] %>"></use></svg>
+    <%= local_assigns['header'] %>
+</h2>
+<% local_assigns['text'] ||= [] %>
+<% local_assigns['text'].each do |text| %>
+    <% if text['type'] == 'large' %>
+        <p class="p-large">
+            <%= text['content'].render_markdown({skip_paragraph_surround: true}) %>
+        </p>
+    <% else %>
```

We can fail fast here by explicitly checking for `type == small` too. Otherwise if someone provides `lrge` as a typo it'll be rendered in small text and it may be difficult to debug:

```
<% if text['type'] == 'large' %>
  <p class="p-large">
    <%= text['content'].render_markdown({skip_paragraph_surround: true}) %>
  </p>
<% elsif text['type'] == 'small' %>
  <%= text['content'].render_markdown %>
<% else 
  raise "Unknown text type: #{text['type']}"
end %>
```